spring-cloud-stream-binders/spring-cloud-stream-binder-rabbit/src/main/resources/META-INF/spring-cloud-stream/rabbit-binder.properties

```diff
@@ -18,6 +18,6 @@ spring.cloud.stream.binder.rabbit.default.prefetch:  1
 spring.cloud.stream.binder.rabbit.default.replyHeaderPatterns:       STANDARD_REPLY_HEADERS,*
 spring.cloud.stream.binder.rabbit.default.republishToDLQ:    false
 spring.cloud.stream.binder.rabbit.default.requestHeaderPatterns:     STANDARD_REQUEST_HEADERS,*
-spring.cloud.stream.binder.rabbit.default.requeue:   true
+spring.cloud.stream.binder.rabbit.default.requeueRejected:   true
 spring.cloud.stream.binder.rabbit.default.transacted:false
-spring.cloud.stream.binder.rabbit.default.txSize:    1
+spring.cloud.stream.binder.rabbit.default.txSize:    1
```

Comment:
I would vote for just "binder" (and use that consistently across binder implementations) since this property maps to a name of a resource that is created within rabbit itself, thus having "rabbit" in the name also, seems redundant. The main purpose is to provide a namespace for the queues/topics we create.